ddtrace/monkey.py

```diff
@@ -29,6 +29,7 @@
     'django': False,
     'flask': False,
     'pylons': False,
+    'botocore': True,
```

Comment:
preserve the alphabetical ordering so move this to the top of the dict. also need `boto: True` as well